tests/regression_tests.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul 14 16:17:57 2020
+
+@author: smullally
+"""
+
+
+import exovetter as vet
+
+
+def test_lpp():
+    """
+    Test reads in an lpp file name, runs the metric without \
+    any plots and returns values.  
+    It then asseerts that the normal one is greater than 0, which must be true.
+    """
+    
+    tce_filename = "tce0001234567_dvt.fits"
+    
+    #Populate the tce class by reading in a file
+    #At this point a phases array, 
+    #and (int) transit_number would get populated.
+   
+    tce = vet.create_tce(file = tce_filename, ext = 1)
+    
+    #This populates the LPP metric fields in this class
+    #It also populates where the plotting data is stored
+    #This function relies on inputs from a precomputed model stored
+    #in something called mapInfo. In theory if someone trains up another 
+    #model we could input it here.
+    tce.run_lpp(mapInfo = None)
```

Comment:
Let's not camel-case. Also, if `map_info` is always required, it should not be an optional keyword, but rather be a positional argument.
```suggestion
    map_info = something_model_thingy
    tce.run_lpp(map_info)
```

Reply:
I don't want users to have to think about this.  it should be possible to call run_lpp without specifying the map_info piece. But I also didn't want to forget that technically the code can be run with a different map.